packages/js/src/settings/components/introduction.js

```diff
@@ -215,6 +215,7 @@ const Introduction = () => {
 
 					{ /* // ----------------- Title and description section of each step -----------------*/ }
 					<div
+						style={ { direction: "ltr" } }
```

Comment:
In general we should not put style on an element directly if we can avoid it. Which made me think why there is no class for this in Tailwind. However, there are the specific `rtl:` prefixes (you could also ternary it because we have the `isRtl` in the JS).
You could solve the same problem with switching the positions in RTL to the other side of the center, like so:
```js
  stepIndex === 0 && "yst--translate-x-0",
  stepIndex === 1 && "yst--translate-x-full rtl:yst-translate-x-full",
  stepIndex === 2 && "yst--translate-x-[200%] rtl:yst-translate-x-[200%]"
```